examples/NeuralDecoding/PositionDecodingFromHippocampus/.bonsai/Bonsai.config

```diff
@@ -0,0 +1,116 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PackageConfiguration xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <Packages>
+    <Package id="Bonsai" version="2.8.5" />
```

Comment:
Please ensure to update this PR to Bonsai 2.9.0 prior to merging, similar to https://github.com/bonsai-rx/machinelearning-examples/pull/18 or https://github.com/bonsai-rx/machinelearning-examples/pull/19

The easiest way to do this is to put the 2.9.0 Bonsai.exe here and launch it and everything else should be automatic. (Don't do it manually, 2.9.0 has new core dependencies it'll need to add.)